Packages/io.chainsafe.web3-unity/Runtime/Scripts/CountlyAnalytics.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ChainSafe.Gaming.Web3.Analytics;
+using Plugins.CountlySDK;
+using Plugins.CountlySDK.Models;
+using UnityEngine;
+
+public class CountlyAnalytics : IAnalyticsClient
+{
+    private const string AppKey = "9fad27b7383f5151a316505ca3245287fc44ba50";
+    private const string ServerUrl = "https://trial-chainsafe.count.ly";
+
+    
+    public async Task CaptureEvent(AnalyticsEvent eventData)
```

Comment:
What if we keep it returning void but make it async? We probably don't need to await this call on the calling site in all cases.

Reply:
We can't do that unfortunately due to the nature of  async modifier.
![image](https://github.com/ChainSafe/web3.unity/assets/8202013/71ae2489-c997-4479-a797-e456355fc019)

Reply:
Left it as a Task.

Reply:
You should be able to remove `async` from the methods signature in the interface, but keep in in the implementation.